File: kotlin-ide/j2k/new/testData/newJ2k/tryWithResource/WithReturnAtEnd.java
//file
import java.io.*;

public class C {
    int foo() {
        try(InputStream stream = new ByteArrayInputStream(new byte[10])) {
            // reading something
            int c = stream.read();
            return c;
        }
        catch (IOException e) {
            System.out.println(e);
            return -1;
        }
    }
}

File: j2k/testData/fileOrElement/tryWithResource/WithReturnAtEnd.kt
import java.io.*

class C {
    internal fun foo(): Int {
        try {
            ByteArrayInputStream(ByteArray(10)).use { stream ->
                // reading something
                return stream.read()
            }
        } catch (e: IOException) {
            println(e)
            return -1
        }

    }
}